var/spack/repos/builtin/packages/r-dada2/package.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class RDada2(RPackage):
+    """Accurate sample inference from amplicon data with single nucleotide
+       resolution"""
+
+    homepage = "http://benjjneb.github.io/dada2/"
+    url      = "https://github.com/benjjneb/dada2/archive/v1.14.tar.gz"
```

Comment:
Any reason not to fetch from bioconductor?

Reply:
Didn't actually know this one was in bioconductor; however, it looks like 1.14 is currently only available from the GitHub.